sources/packages/web/src/views/aest/ApplicationDetails.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Student Details</a
+    >
+  </h5>
+  <full-page-container v-if="applicationDetail.data">
+    <h2 class="color-blue padding-bottom-30">
+      Student Application Details
+      {{
+        applicationDetail.applicationNumber
+          ? " - " + applicationDetail.applicationNumber
+          : ""
+      }}
+    </h2>
+    <formio
+      :formName="selectedForm"
+      :data="initialData"
+      :readOnly="true"
+      @loaded="formLoaded"
+    ></formio>
+    <v-row>
+      <v-col md="6">
```

Comment:
I feel this is a kind of duplication with 
![image](https://user-images.githubusercontent.com/77353155/140802774-fe0e97a7-6363-4e16-9a73-14d0ab9b93eb.png), maybe we can make the form as a separate component and call here in read-only mode? thoughts @devs? just a thought

Reply:
I felt it would involve some significant work to make this component as a common(FullTimeVue) and also considering the code refactoring that is done with this story like ApplicationStatus, StudentPD banner. I thought this work can be done as a technical debt to make it one component in Student, Ministry and wherever required.

Reply:
Maybe we can do it now or later but I do not see how StudentPD or ApplicationStatus could impact this decision. Maybe I am missing something.

Reply:
Issue #761 has been created to address the Student Application dropdown fix and organize the APIs after discussing with @abschwenker

Reply:
Just to be clear, the ticket is not about fixing dropdowns, it is about creating a common component to be shared among the Student and the Ministry portal, that yes, will fix the dropdowns, but also fixed everything else that is needed 😉